rainier-core/src/main/scala/com/stripe/rainier/ir/Packer.scala

```diff
@@ -29,6 +29,10 @@ private class Packer(methodSizeLimit: Int) {
 
   private def traverseIR(p: IR): (IR, Int) =
     p match {
+      case s: SumIR =>
+        val size = s.exprs.size
```

Comment:
when the `exprs` are all `Ref` this can just be `(s, s.exprs.size)`

Reply:
With a call to traverse first?

Reply:
No, there's no need to traverse the Refs here because there's no chance of splitting them up and their size is known to be 1.

Reply:
Ah, okay, cool.